shortfin/CMakeLists.txt

```diff
@@ -39,6 +39,10 @@ if(NOT WIN32)
   set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 endif()
 
+# For unicode support Windows libfmt requires compiling with /utf-8.
+add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
+add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
```

Comment:
I take it these can't be added here, since they need to apply to `fmt::fmt` too?
https://github.com/nod-ai/shark-ai/blob/d08b0e5dc4ecf4af364121933fbe0aa61031cc66/shortfin/build_tools/cmake/shortfin_library.cmake#L7-L17

Reply:
Sure, I give it a try.

Reply:
Didn't seem to work: https://github.com/nod-ai/shark-ai/actions/runs/12379929075/job/34555169307?pr=711#step:10:593. Keeping it at the top level is fine I guess. We try to keep options scoped as tightly as possible. At least https://cmake.org/cmake/help/latest/command/add_compile_options.html applies to the current directory and below, so downstream users shouldn't also need to set this... I think.

Reply:
Yeah, I just moved it back to the top level.